sqre-uservice-{{cookiecutter.svc_name}}/kubernetes/uservice-{{cookiecutter.svc_name}}-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: u-{{ cookiecutter.svc_name }}
+  labels:
+    name: u-{{ cookiecutter.svc_name }}
+spec:
+  type: NodePort
+  ports:
+  - name: http
+    port: 5000 # FIXME: must not conflict with any other services
```

Comment:
At the moment I do not see one.  We have to know which services are running on a given GKE cluster and tweak appropriately.  This does indeed seem grimly suboptimal.  Perhaps I have missed something in either the Kubernetes or GKE docs.